extensions/amp-analytics/0.1/test/test-config.js

```diff
@@ -882,6 +883,86 @@ describes.realWin('AnalyticsConfig', {amp: false}, env => {
     });
   });
 
+  describe('warning message', () => {
+    it('shows the warning', () => {
+      ANALYTICS_CONFIG['-test-venfor'] = {
+        'requestOrigin': 'https://example.com',
```

Comment:
nit: 'requestOrigin' is not required. Let's remove it to save one line.

Reply:
Done